RR.App/Controllers/HRIS/RoleManageController.cs

```diff
@@ -9,6 +11,7 @@ namespace RR.App.Controllers.HRIS;
 [ApiController]
 public class RoleManageController : ControllerBase
 {
+    private readonly AuthorizeIdentity _identity;
     private readonly IRoleAccessLinkService? _roleAccessLinkService;
     private readonly IRoleAccessService? _roleAccessService;
     private readonly IRoleService? _roleService;
```

Comment:
```suggestion
    private readonly IRoleAccessLinkService _roleAccessLinkService;
    private readonly IRoleAccessService _roleAccessService;
    private readonly IRoleService _roleService;
```

I'm not sure why these services are marked as nullable - Should never be nullable. Please change this as well.